sos/report/plugins/kernelrt.py

```diff
@@ -36,6 +36,12 @@ def setup(self):
         # note: rhbz#1059685 'tuna - NameError: global name 'cgroups' is not
         # defined this command throws an exception on versions prior to
         # 0.10.4-5.
-        self.add_cmd_output('tuna -CP')
+        tuna_cmd = 'tuna'
+        co = {'cmd': '%s --help' % tuna_cmd, 'output': 'show_threads'}
+        if self.test_predicate(self, pred=SoSPredicate(self, cmd_outputs=co)):
+            tuna_cmd += ' show_threads -C'
+        else:
+            tuna_cmd += ' -CP'
```

Comment:
Ack to the intent, but we're trying to move off of legacy string interpolation methods and to f-strings globally. We could do this and button it up a bit with something like:

```python
co = {'cmd': 'tuna --help', 'output': 'show_threads'}
self.add_cmd_output(
    f"tuna {'show_threads -C' if bool(SosPredicate(self, cmd_outputs=co)) else 'CP'}"
)
```

Reply:
Ack, this looks really elegant. I'll make the change and push again.

Reply:
I moved the predicate test outside of the f-string because when I tried to break the curly braces and couldn't find a clean way. As always, any suggestion is welcome!